pkg/deploy/controller.go

```diff
@@ -75,11 +74,7 @@ type watcher struct {
 // start calls listFiles at regular intervals to trigger application of manifests that have changed on disk.
 func (w *watcher) start(ctx context.Context, client kubernetes.Interface) {
 	nodeName := os.Getenv("NODE_NAME")
-	broadcaster := record.NewBroadcaster()
-	broadcaster.StartLogging(logrus.Infof)
-	broadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: client.CoreV1().Events(metav1.NamespaceSystem)})
-	w.recorder = broadcaster.NewRecorder(schemes.All, corev1.EventSource{Component: ControllerName, Host: nodeName})
-
+	w.recorder = pkgutil.BuildControllerEventRecorder(client, ControllerName, nodeName)
```

Comment:
lets just move the `os.Getenv("NODE_NAME")` call in there too since we're pretty consistent about setting that.

Reply:
fixed